src/components/Footer/Footer.tsx

```diff
@@ -34,8 +35,19 @@ export const Footer: React.FC = () => {
 		? socialMediaDataLogIn
 		: socialMediaDataLogOut;
 
+	const NoHeaderRoutes = [
```

Comment:
why name start from upper case

Reply:
move above the component, we dont need reinitialisation with each rerender

Reply:
Deleted and now i use another logic.